plugins/woocommerce/readme.txt

```diff
@@ -1,5 +1,5 @@
 === WooCommerce ===
-Contributors: automattic, mikejolley, jameskoster, claudiosanches, rodrigosprimo, peterfabian1000, vedjain, jamosova, obliviousharmony, konamiman, sadowski, wpmuguru, royho, barryhughes-1
+Contributors: automattic, woocommerce, mikejolley, jameskoster, claudiosanches, rodrigosprimo, peterfabian1000, vedjain, jamosova, obliviousharmony, konamiman, sadowski, wpmuguru, royho, barryhughes-1, claudiulodro, tiagonoronha, ryelle, levinmedia, aljullu, nerrad, joshuawold, assassinateur, haszari, mppfeiffer, nielslange, opr18, ralucastn, tjcafferkey, danielwrobert, patriciahillebrandt, albarin, dinhtungdu, imanish003, karolmanijak, sunyatasattva, alexandrelara, gigitux, danieldudzic, samueljseay, alexflorisca, opr18, tarunvijwani, pauloarromba, saadtarhi
```

Comment:
Maybe I'm being pedantic, but now this list will still look very incomplete, not including many contributors who work on wc-admin and other parts where basically everything is part of core now?

Just food for thought.

Reply:
Thanks for reviewing this proposed change and for mentioning that not all contributors have been added, @samueljseay. For [WooCommerce Beta Tester](https://github.com/woocommerce/woocommerce-beta-tester), I'll add all contributors as mentioned as mentioned on https://github.com/woocommerce/woocommerce-beta-tester/blob/c0d20973cc021828f32600187ec5566eca468529/readme.txt#L2. For [WooCommerce Admin](https://github.com/woocommerce/woocommerce-admin), we don't have such a file that lists all contributors. We could add the contributors that are listed on https://github.com/woocommerce/woocommerce-admin/graphs/contributors. However, I wonder if we should define a minimum number of contributions to be added to the list. For Woo AI and WooCommerce Docs, we don't have designated repos. Finding out who contributed to these products, might require a lot of manual effort. What are your thoughts on that?

Reply:
@nielslange I wonder if we could just add the active developer members of all woo teams as defined in github? I realize this won't catch community contributors but maybe its better and doesn't require a lot of manual work?

That said, it's just my opinion of one possible approach 😅

Reply:
This would be an option, @samueljseay. Unfortunately, I don't have the necessary permissions to access the `Collaborators and teams` page within the `Settings` section of this repo. I could go through https://github.com/orgs/woocommerce/teams, but that also seems pretty time-consuming, as I need to double-check whether the corresponding team is contributing to Woo core or not.

Reply:
Can we limit the scope of this PR to our teams, and ping others to do the same?

Reply:
I think that would be the best approach, @wavvves, as adding all contributors requires a lot of manual work.

Reply:
good idea @wavvves 👍🏻

Reply:
Thanks for sharing your thoughts, @samueljseay and @wavvves. I'll mark this conversation as resolved for now.